### ISSUE TITLE
[Android] Add Stickers to 2nd WhatsApp Account

### ISSUE THREAD

**Issue by santi-petersen** [WhatsApp/stickers#1005]
**Specifications**  
OS Version: Probably all versions that allow more than one WhatsApp account
Device Manufacturer: All

**Problem Description**  
Since WhatsApp recently added a feature that allows you to have 2 accounts and 2 phone numbers on the same phone, it is neccesary now any way to select in what account the stickers will be added, or it must add them in both accounts. At the moment stickers are added only in the 1st account.

**Reproduction Steps**  
Add a second account to WhatsApp and try to add the pack to that account.

**Comment by Korogba** [comment]
> it is neccesary now any way to select in what account the stickers will be added, or it must add them in both accounts. At the moment stickers are added only in the 1st account

I just tested this and adding works even after I switch accounts. Is the expectation here that adding a pack to the active account should also add it to the inactive account?

**Comment by santi-petersen** [comment]
OK. I verified that the pack is added always on the active account. **I'm not sure what the best behavior is**, whether to keep adding packs separately or share packs between accounts.

Maybe it's a topic of discussion. One of the users of my app uses two accounts and he says that when he wants to send a sticker he never find it, because he has many packs on both accounts and it is confusing. After spending some time searching for the pack and discovering that it's not there, he need to open the app, find the pack, and add it "again" to WhatsApp.

In my opinion, I think sharing all packs on both accounts is better behavior. And at least that's what this user expects.

**Comment by Korogba** [comment]
> In my opinion, I think sharing all packs on both accounts is better behavior. And at least that's what this user expects.

yeah, I could bring this up internally but given the accounts are separate and usually do not share any other information, I am not sure about this. One option might be to give the user the additional option of adding to the second account

**Comment by santi-petersen** [comment]
> One option might be to give the user the additional option of adding to the second account

Yeah, that is even a better solution. I understand that changes must be made in WhatsApp, not in this app. For example, the user can be asked in the dialog that WhatsApp displays when the user agree or not to add the pack.

**Comment by Muhammad-Rizwan804** [comment]
how can i add the my own custom sticker pack in the what's app in android kts
